### ISSUE TITLE
Raid for LoA

### ISSUE THREAD

**Issue by freeWilly77** [psouza4/EKUnleashed#5]
Can someone make raid/Lilith working for LoA? I would appreciate it very much

And maybe also do dungeon automatically

**Comment by zs0** [comment]
Very much agreed. LoA support is needed. It's also a shame that the latest update did not work for LoA. :(

**Comment by psouza4** [comment]
We need someone familiar with LoA to do this.  I don't play that game, personally.  URLs, packet sniffs, etc. would be handy.

Also, @zs0: github isn't a discussion forum, so please let's not digress a bug report or feature request into chat, however rest assured that I'm working on individual deck allocation support for LoA as we speak.  I wish Erias had done this for all game services instead of only EK, but people were asking for the changes he added immediately.

**Comment by zs0** [comment]
Understood. Is there a better way to agree with an open issue to display user support for it? Or was my comment about the update the unnecessary part?

Thanks for putting in the extra time for LoA. I wish I knew more to assist you with URL's, etc. etc. but it seems I am a bit naive with this stuff at the moment.

**Comment by psouza4** [comment]
@zs0: there's a forum at www.EKUnleashed.com you can join, although I don't have a lot of time to participate in discussions currently.  It really needs a few people to step up and become moderators so the community can thrive.  There's also a LINE group chat with several players and developers (including myself) that discuss EKU exclusively.

**Comment by zs0** [comment]
Yep I frequent that forum but it seemed as if this was the new proper place to check in on. I would love to join the LINE chat. Thank you Pete.

**Comment by psouza4** [comment]
@zs0: my user ID on LINE is also psouza4 as it is here at github.  Please feel free to message me for a LINE group chat invite.
